tslint -t vso ./src/**/*.ts   fails  on running 'yo botbuilder-enterprise'

## Project

**Name**: Enterprise Bot Template

**Language**:  TypeScript

## Description

Running 'yo botbuilder-enterprise' fails at below step:   

tslint -t vso ./src/**/*.ts

With this error attached:
[error.txt](https://github.com/Microsoft/AI/files/3084422/error.txt)


npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! yee-bot@1.0.0 lint: `tslint -t vso ./src/**/*.ts`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the yee-bot@1.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.



## To Reproduce
run  'yo botbuilder-enterprise'  to create a new enterpris_bot project

## Expected behavior
Error should not be encountered

## Screenshots
<!--If applicable, add screenshots to help explain your problem.-->

## Additional context
<!--Add any other context about the problem here.-->

--- a/templates/Enterprise-Template/src/typescript/enterprise-bot/tslint.json
+++ b/templates/Enterprise-Template/src/typescript/enterprise-bot/tslint.json
@@ -17,7 +17,8 @@
       ],
       "linebreak-style": [false],
       "member-ordering": [false],
-      "no-relative-imports": [false]
+      "no-relative-imports": [false],
+      "completed-docs": [false]
     },
     "rulesDirectory": [
       "node_modules/tslint-microsoft-contrib"

--- a/templates/Enterprise-Template/src/typescript/generator-botbuilder-enterprise/generators/app/templates/enterprise-bot/tslint.json
+++ b/templates/Enterprise-Template/src/typescript/generator-botbuilder-enterprise/generators/app/templates/enterprise-bot/tslint.json
@@ -17,7 +17,8 @@
       ],
       "linebreak-style": [false],
       "member-ordering": [false],
-      "no-relative-imports": [false]
+      "no-relative-imports": [false],
+      "completed-docs": [false]
     },
     "rulesDirectory": [
       "node_modules/tslint-microsoft-contrib"

--- a/templates/Skill-Template/src/typescript/generator-botbuilder-skill/generators/app/templates/customSkill/tslint.json
+++ b/templates/Skill-Template/src/typescript/generator-botbuilder-skill/generators/app/templates/customSkill/tslint.json
@@ -17,7 +17,8 @@
       ],
       "linebreak-style": [false],
       "member-ordering": [false],
-      "no-relative-imports": [false]
+      "no-relative-imports": [false],
+      "completed-docs": [false]
     },
     "rulesDirectory": [
       "node_modules/tslint-microsoft-contrib"

